src/block-management/block-manager.js

```diff
@@ -123,9 +123,6 @@ export class BlockManager extends React.Component<PropsType, StateType> {
 			const insertionIndex = indexAfterSelected || this.props.blockCount;
 			this.props.insertBlock( newBlock, insertionIndex );
 		}
-
-		// now set the focus
-		this.props.selectBlock( newBlock.clientId );
```

Comment:
After removing this line, do we still need the `selecBlock` prop for anything? Should we remove all references from the file?

Reply:
Good catch, we don't. I'll remove it. 🙇

Reply:
Done with 4e264fa.